gengokit/httptransport/httptransport.go

```diff
@@ -107,11 +107,18 @@ func NewBinding(i int, meth *svcdef.ServiceMethod) *Binding {
 			newField.GoType = "[]" + newField.GoType
 		}
 
+		// IsEnum needed for ConvertFunc and TypeConversion logic just below
+		newField.IsEnum = field.Type.Enum != nil
 		newField.ConvertFunc, newField.ConvertFuncNeedsErrorCheck = createDecodeConvertFunc(newField)
 		newField.TypeConversion = createDecodeTypeConversion(newField)
 
 		nBinding.Fields = append(nBinding.Fields, &newField)
 
+		// Enums are allowed in query/path parameters, skip warning
```

Comment:
If Enums are allowed as both query and path parameters, we should have a test for the path parameters.

I see a test for use as query parameter, though not one for the path parameter.

We don't need to support it in the path currently, though unless we have a test we should warn about it.